src/playground/components/AlertsActionBar.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+export default function AlertsActionBar({
+	messages,
+	onFixAll,
+	options,
+	onUpdate,
+	rulesWithInvalidConfigs,
+	setRulesWithInvalidConfigs,
+}) {
+	return (
+		<div className="alerts-action-bar">
+			<div className="alerts-action-bar__content">
+				<span>Line No.</span>
+				<span>Description</span>
```

Comment:
I think line no. and description is already provided by the alerts below this one so instead of this we can provide a short description of this alert button something like:

>Fix all the issues or Disable the rules with issues.